package.json

```diff
@@ -22,7 +23,8 @@
         "react": "^16.8.6",
         "react-dom": "^16.8.6",
         "react-intl": "^2.9.0",
-        "react-select": "^2.0.0"
+        "react-select": "^2.0.0",
```

Comment:
Thanks, good to know. I had a suspicion that's where it was coming from based on the error message, but since it was just a warning I didn't want to risk breaking a ton of stuff.

Reply:
Or move onto https://material-ui.com/components/autocomplete/